lib/prepare_security.js

```diff
@@ -70,7 +70,7 @@ export default class PrepareSecurityRelease extends SecurityRelease {
         { cli: this.cli, repository: this.repository }
       );
     }
-    this.cli.info(`Merge pull request with:
+    this.cli.info(`If the PR is ready (CI is green): merge pull request with:
```

Comment:
nit: we probably accept yellow CIs as well
```suggestion
    this.cli.info(`If the PR is ready (CI is passing): merge pull request with:
```